lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -859,12 +854,13 @@ def install_puppetlabs_release_repo_on( hosts, repo = nil, opts = options )
             opts = FOSS_DEFAULT_DOWNLOAD_URLS.merge(opts)
 
             case variant
-            when /^(fedora|el|centos)$/
+            when /^(fedora|el|centos|sles)$/
```

Comment:
should this be an issue for the variant?  This should still be `osx` in this case, while `elcapitan` should be the codename:

    variant, version, arch, codename = host['platform'].to_array

When I hit about on my mac, I see "OS X El Capitan", which leads me to believe that this shouldn't be an issue for the variant.  `elcapitan` wouldn't pass our platform string validation as a platform name, are there team's using it as such?